### PR TITLE
{lang,tools}[foss/2017a] Node-RED v0.16.2, nodejs v6.10.3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/Node-RED/Node-RED-0.16.2-foss-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+version = '0.16.2'
+
+homepage = 'http://nodered.org'
+description = """Node-RED is a programming tool for wiring together hardware devices, APIs and online services
+ in new and interesting ways."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = ['https://github.com/node-red/node-red/releases/download/%(version)s/']
+sources = ['node-red-%(version)s.zip']
```
Comment (wpoely86) [inline easybuild/easyconfigs/n/Node-RED/Node-RED-0.16.2-foss-2017a.eb]: `%(namelower)s` ?

File: easybuild/easyconfigs/n/nodejs/nodejs-6.10.3-foss-2017a.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ event-driven, non-blocking I/O model that makes it lightweight and efficient, 
+ perfect for data-intensive real-time applications that run across distributed devices."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'lowopt': True}
+
+source_urls = ['http://nodejs.org/dist/v%(version)s/']
+sources = ['node-v%(version)s.tar.gz']
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
```
Comment (wpoely86) [inline easybuild/easyconfigs/n/nodejs/nodejs-6.10.3-foss-2017a.eb]: why system?

Comment (boegel) [inline easybuild/easyconfigs/n/nodejs/nodejs-6.10.3-foss-2017a.eb]: it needs any Python as a build dependency, and we know there is one...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2123.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5bc04b23efe462297eb94e705a92b220 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2555.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5fb0ddffc1d7a77416bb316b84e7f5e4 for a full test report.

File: easybuild/easyconfigs/n/nodejs/nodejs-6.10.3-foss-2017a.eb (hunk 2 above)
Comment (wpoely86) [inline easybuild/easyconfigs/n/nodejs/nodejs-6.10.3-foss-2017a.eb]: add that as a comment?

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**